src/controllers/cart.controller.js

```diff
@@ -0,0 +1,45 @@
+const cartService = require("../services/cart.service");
+
+const createCart = async (req, res) => {
+  try {
+    const { productoptionId, quantity } = req.body;
+    const userId = req.user.id;
+
+    await cartService.createCart(productoptionId, quantity, userId);
+
+    res.status(201).json({ message: " Cart Created!" });
```

Comment:
```suggestion
    return res.status(201).json({ message: " Cart Created!" });
```
response를 보내고 더이상 처리할 로직이 없다면 return 해주세요